### DISCUSSION TITLE
This extension is broken, don't bother trying to use it

### DISCUSSION THREAD

**Discussion by levicki** [d8ahazard/sd_dreambooth_extension#1411] [General]
This extension is broken, don't bother trying to use it

**Comment by d8ahazard** [comment]
Hey, that's super helpful and totally accurate. Thanks for the massive contribution.

**Reply by levicki** [reply]
@d8ahazard At least creating this discussion topic got you to read it, maybe you will even read this response?

I just tried using the extension again to train a LoRA and it still doesn't work properly &mdash; you can't complete or continue training despite everything being installed correctly.

I encountered two issues:

1. Someone says there should be a check for SDXL model type [here](https://github.com/d8ahazard/sd_dreambooth_extension/blob/1cf7267f49ecfe24fec9756b1d76c0ad8629087f/dreambooth/train_dreambooth.py#L461) &mdash; adding it resolves the exception and allows the training of LoRA from a SD1.x model to start.
2. The other exception is caused by failing to rename a file and it's coming from [here](https://github.com/d8ahazard/sd_dreambooth_extension/blob/1cf7267f49ecfe24fec9756b1d76c0ad8629087f/dreambooth/diff_lora_to_sd_lora.py#L92) &mdash; It fails to rename `_auto.safetensors` file somehow creating a zombie original `.safetensors` file in the process which can only be deleted after a full reboot despite no open handles being held on it by any userspace process. What is suprising that there's no exception on deleting it before rename fails.

I did try to [contribute](https://github.com/d8ahazard/sd_dreambooth_extension/issues/1352) earlier, but it went nowhere because bug report was labeled by @saunderez as a question even though I insisted it was a bug, then by your overzealous bot as stale, and then it got closed without anyone taking a stab at it or at least outright stating that behavior is intended or won't be changed.

I am well aware that this is open-source and that nobody owes me anything and as such my expectations should be low and they are the lowest possible &mdash; triaging a reported issue and acknowledging it if it is reproducible. Also, unlike many others reporting issues in open-source repos I am aware that people reporting issues are expected to contibute (sometimes with PRs because maintainers are busy), but the way the issue I reported was handled wasn't exactly encouraging hence this "massive contribution" here.

That said, if you actually want me to contribute with proper bug reports and logs do let me know.
